### PR TITLE
Add SwiftPM manifest

### PR DESCRIPTION
**Pull request by MatkovIvan**
[AB#79780](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/79780)

## Changes aside of `Package.swift`

- Fix to force avoid adding prefix to symbols even for protobuf-c files because prefix headers are not supported;
- Disabled 64-bit mach_* APIs because it cannot generated required stuff from system `mach_exc.defs` file; See [this](https://www.mikeash.com/pyblog/friday-qa-2013-01-11-mach-exception-handlers.html) article for details

### CODE REVIEW

File: Source/PLCrashMacros.h [hunk 1]
```diff
@@ -42,7 +42,10 @@
 #endif
 
 #if defined(__cplusplus)
-#  if defined(PLCRASHREPORTER_PREFIX)
+#  define NO_OTHER_MACRO_STARTS_WITH_THIS_NAME_
+#  define IS_EMPTY_(name) defined(NO_OTHER_MACRO_STARTS_WITH_THIS_NAME_ ## name)
+#  define IS_EMPTY(name) IS_EMPTY_(name)
```
Comment (annakocheshkova) [inline Source/PLCrashMacros.h:47]: Why did you need two macros for this?

Comment (MatkovIvan) [reply Source/PLCrashMacros.h:47]: To enable multi pass preprocessor magic 😄
